flux-middle/src/fhir.rs

```diff
@@ -285,7 +287,7 @@ pub struct AdtDef {
     pub refined_by: RefinedBy,
     pub invariants: Vec<Expr>,
     pub opaque: bool,
-    sorts: Vec<Sort>,
+    pub(crate) sorts: Vec<Sort>,
```

Comment:
Do we still need this visibility? This was purposely hidden because it's kind of like an optimization since the sort information is already present in `refined_by`

```suggestion
    sorts: Vec<Sort>,
```